rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,435 @@
+
+#### Roadmap Retrospective
+
+The Roadmap Retrospective occurs during the recurring Project Leads meeting and not in a special session. The Roadmap Shepherd MUST add the Retrospective to the agenda and faciliate.
+
+In preparation, the Project Leads MUST independently review the prior year’s Product Roadmap, Release Retrospective summaries, and appropriate supporting documentation and then share their significant findings and learnings with their peers during the Retrospective.
+
+Findings may include: Themes that were incomplete and the accompanying reason, incorrectly sized work in the Engineering Plan, etc. These findings SHOULD be utilized as a reference class for the next Product Roadmap.
+
+Project Leads MUST identify a Roadmap Shepherd during the Retrospective. The Shepherd MUST NOT be a Project Lead, to ensure the neutrality of the content for the Roadmap.
+
+The Shepherd is responsible for:
+* Managing the timeline for the creation of the draft Roadmap
+* Facilitating all meetings related to the creation of the draft Roadmap 
+* Documenting important decisions, discussions, and questions 
+* Assigning action items to move toward the creation of the Roadmap, should no Project Lead volunteer, and following up with them through completion
+
+### Drafting the Roadmap
+[Drafting the Roadmap]:#drafting-the-roadmap
+
+Project Leads MUST independently create Themes by utilizing the input provided in the Science Roadmap, Technical Objectives RFC, DCP User Survey and UX Insights, and Roadmap Retrospective.
+
+To create the draft Product Roadmap, the Roadmap Shepherd:
+- Forks the *HumanCellAtlas dcp-community* repository
+- Copies `rfcs/roadmap-template.md` to `rfcs/text/0000-roadmap-YYYY.md` - where "YYYY" is the target year for the roadmap. 
+
+The Roadmap Shepherd MUST collate all themes into the draft which is an Informational RFC. 
+
+Project Leads MUST review the Themes as a collective and distill the revised list to a lesser number of themes. This is achieved through a consensus building process, which is implemented by the Roadmap Shepherd and occurs during specifically noted Project Lead meetings, outlined below:
+
+* Explore the Themes, search for commonalities, and document concerns and issues 
+* Identify potential consolidated or alternative Themes, resolution for concerns and issues, and action items required for further understanding
+* Prioritize the Themes within the Roadmap
+* Update the Roadmap to reflect revised content
+* Resolve outstanding questions and concerns
+
+If the Project Leads fail to reach consensus within the specified timebox, then the draft MUST be escalated to Oversight for consultation.
+
+When rough consensus is reached, the Roadmap is submitted as a pull request to the *dcp-community* repo and follows the DCP RFC process for a *Governance* RFC starting with [community review](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0001-rfc-process.md#reviewing-rfcs). 
```

Comment:
"rough consensus" should be defined

Reply:
I can add a reference to [On Consensus and Humming in the IETF](https://tools.ietf.org/html/rfc7282).

Reply:
Addressed in 3ff580804dc32ecb6beb54ac661fab36db67981f